src/resolver.ts

```diff
@@ -4,11 +4,13 @@ export const resolve = (appFnId: string, opts?: ResolveOptions) => {
   opts = opts || defaultOptions
   // These are mostly to ease testing
   const basedir = opts.basedir || process.cwd()
-  const resolver = opts.resolver || require('resolve').sync
+  const resolver: Resolver = opts.resolver || require('resolve').sync
```

Comment:
Could `require('resolve').sync` get converted to `import { sync } from 'resolve'` and moved to the top?

Reply:
Absolutely. Done 👍